queries/ecma/indents.scm

```diff
@@ -62,7 +62,8 @@
 (parenthesized_expression
   ("("
     (_)
-    ")" @indent.end))
+    ")" @indent.end)
+  !condition)
```

Comment:
Wait what 🤔 

Isn't parenthesized expression the condition itself, how can it check for whether it contains a condition field or not

Reply:
The syntax is weird, I think to check against
```query
anythingbutthis: (node)
```
you use
```query
(node !anythingbutthis) @yay
```

Reply:
Yeah, after rereading the docs I think I misunderstood the negated fields part completely